app/jobs/waste_notification_job.rb

```diff
@@ -18,11 +18,14 @@ def self.perform(waste_registration_id, waste_pickup_time_id)
     message_options = { title: title, body: body }
     messages = [message_options.merge(to: device.token)]
 
-    # sending message
+    # Sending message with rescuing errors
     client = Exponent::Push::Client.new
-    feedback = client.send_messages(messages)
 
-    # Log PushNotification
-    RedisAdapter.add_push_log(device.token, message_options.merge(date: DateTime.now, payload: feedback.try(:response).try(:body)))
+    begin
+      feedback = client.send_messages(messages)
+      RedisAdapter.add_push_log(device.token, message_options.merge(date: DateTime.now, payload: feedback.try(:response).try(:body)))
+    rescue => e
+      RedisAdapter.add_push_log(device.token, message_options.merge(rescue_error: "waste push notification", error: e, date: DateTime.now))
```

Comment:
wollen wir hier vllt noch zusätzlich was zu Rollbar schicken?

```
# with a description
Rollbar.error(e, 'The user info hash doesn\'t contain the correct data')

# with extra data giving more insight about the exception
Rollbar.error(e, :user_info => user_info, :job_id => job_id)
```

Reply:
wir haben ja die PushLogs mit den Fehler schon, denke das sollte erstmal reichen, oder?

Reply:
ok. wusst nicht genau wie die push logs ankommen und wir die mitkriegen. rollbar wird uns ja den fehler jetzt mit rescue nicht mehr melden, richtig?!

Reply:
man kann Rollbar ja auch in den Rescue Block dazu packen, glaube aber es ist erstmal nicht notwendig

Reply:
und wir würden gefahr laufen zu viele rollbar meldungen zu triggern und dann limits auszureizen